packages/e2e-tests/scripts/save-examples.ts

```diff
@@ -71,7 +71,7 @@ function logAnimated(text: string): () => void {
 async function saveRecording(example: string, recordingId?: string) {
   if (recordingId) {
   } else {
-    const recordings = listAllRecordings();
+    const recordings = listAllRecordings({ directory: "/root/.replay" });
```

Comment:
We probably don't want to commit this, I think it will break running this script on a laptop.